src/style.css

```
@@ -1 +1,22 @@
 /* styles go here */
+.fieldset:not(:first-child) {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
```

Margin top is redundant, when u add both appears margin [collapse](https://www.freecodecamp.org/news/what-is-margin-collapse-and-how-to-avoid-it/)

```suggestion
.fieldset {
  margin-bottom: 20px;
}
```